From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Sun, 05 Jul 2026 20:19:20 +0000
Subject: [PATCH] Fenix - Remove Firefox Sync bookmark promotion

Removes Mozilla's promotion to enable Firefox Sync at the bookmarks fragment.

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/bookmarks/BookmarksScreen.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/bookmarks/BookmarksScreen.kt
index 58434eee1781..8e1209cee015 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/bookmarks/BookmarksScreen.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/bookmarks/BookmarksScreen.kt
@@ -1027,7 +1027,7 @@ private fun EmptyList(
             when (state) {
                 is EmptyListState.NotAuthenticated -> RootEmptyContent(
                     dispatcher,
-                    showSignIn = true,
+                    showSignIn = false,
                 )
                 EmptyListState.Authenticated -> RootEmptyContent(
                     dispatcher,
-- 
